drivers/gpu/drm/i915/i915_sysfs.c

```diff
@@ -487,19 +486,25 @@ void i915_gem_remove_sysfs_file_entry(struct drm_device *dev,
 	}
 	mutex_unlock(&dev->filelist_mutex);
 
+	mutex_lock(&drm_global_mutex);
 	if (open_count == 1) {
-		struct i915_gem_file_attr_priv *attr_priv;
+		struct i915_gem_file_attr_priv *attr_priv = file_priv->obj_attr->private ;
```

Comment:
should move it under the following check:
if ((file_priv->obj_attr == NULL) || (attr_priv == NULL)) {

Reply:
Good point